typescript/test/testdata/texttospeech/src/v1/text_to_speech_client.ts.baseline

```diff
@@ -313,4 +313,4 @@ export class TextToSpeechClient {
     return this._innerApiCalls.synthesizeSpeech(request, options, callback);
   }
 
-}
```

Comment:
Please add new lines to the template and baseline files.
![image](https://user-images.githubusercontent.com/4015807/67604675-120fe300-f731-11e9-93ed-3b0a4fa5ab23.png)